Header header
float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw